src/PerfView.Tests/StackViewer/StackWindowTests.cs

```diff
@@ -37,7 +37,7 @@ public Task TestIncludeItemOnByNameTabAsync()
             return TestIncludeItemAsync(KnownDataGrid.ByName);
         }
 
-        [WpfFact]
+        [WpfFact(Skip = "Failing with indexOutOfRange.  See issue https://github.com/Microsoft/perfview/issues/354")]
```

Comment:
:bulb: Overall this was handled great given I wasn't available for responding. For Roslyn we don't bother with a longer message since the issue will already contain it, so this would be considered sufficient:

```csharp
[WpfFact(Skip = "https://github.com/Microsoft/perfview/issues/354")]
```